alpaca/rest.go

```diff
@@ -742,6 +744,121 @@ func (c *Client) GetAsset(symbol string) (*Asset, error) {
 	return &asset, nil
 }
 
+type GetOptionContractsRequest struct {
+	UnderlyingSymbols string           `json:"underlying_symbols"`
+	ShowDeliverable   *bool            `json:"show_deliverables"`
+	Status            *OptionStatus    `json:"status"`
+	ExpirationDate    *civil.Date      `json:"expiration_date"`
+	ExpirationDateGTE *civil.Date      `json:"expiration_date_gte"`
+	ExpirationDateLTE *civil.Date      `json:"expiration_date_lte"`
+	RootSymbol        *string          `json:"root_symbol"`
+	Type              *OptionType      `json:"type"`
+	Style             *OptionStyle     `json:"style"`
+	StrikePriceGTE    *decimal.Decimal `json:"strike_price_gte"`
+	StrikePriceLTE    *decimal.Decimal `json:"strike_price_lte"`
+	Limit             *int             `json:"limit"`
+	PPInd             *bool            `json:"ppind"`
```

Comment:
Recently I started not using pointers to optional fields for two main reasons:
1. It's much more comfortable to use:
``` go
showDeliverable := true
req := GetOptionContractsRequest{ShowDeliverable: &showDeliverable}
```
vs
``` go
GetOptionContractsRequest{ShowDeliverable: true}
```
2. There is no meaningful difference between a missing field vs a field with zero value.